### DISCUSSION TITLE
Instruction for Configuration in HOOBS

### DISCUSSION THREAD

**Discussion by nrtkin** [iRayanKhan/homebridge-tuya#286] [New device help]
Hello,
how should the configuration be done?
First i installed the Tuya-LAN and than i read in youTube that i have to install this plugin in HOOBS.
But i don't know, how and where to write the Device ID and Key?
Can you describe a bit more where to write the device ID's?

Thank you

The config.json looks at the moment like this:

![0001](https://user-images.githubusercontent.com/84382523/118636191-b60a2500-b7d4-11eb-827a-5fca4dfcceb3.png)

**Comment by iRayanKhan** [comment]
Use the official homebridge UI.
